dashboard/app/models/levels/blockly.rb

```diff
@@ -462,6 +463,27 @@ def localized_authored_hints
     end
   end
 
+  # Can apply translations to blockly block XML which is embedded in rich text such as Markdown
+  # strings.
+  # For example:
+  # "Esto es un <xml><block>block</block></xml>." -> "Esto es un <xml><block>bloque</block></xml>."
+  # @param text [String] Text which might have blockly XML embedded in it and needs localization.
+  # @return [String] Text with localized blockly blocks.
+  def localized_blockly_in_text(text)
+    return unless text
+    # Surround the text in a <root /> node so we can parse it as an XML document.
+    text_xml_doc = Nokogiri::XML("<root>#{text}</root>")
+    # Selects the `text` XML elements which will be inside the <xml><root>{text}</root></xml>
+    text_xml = text_xml_doc.child.children
```

Comment:
Would it be beneficial to explicitly get the root here? Maybe either using `.root` or another method?

Reply:
Nice, I didn't know I could do that. That is much clearer.